.travis.yml

```
@@ -13,6 +13,15 @@ matrix:
   fast_finish: true
 notifications:
   email: false
+jobs:
+  include:
+    - stage: "DocTest"
+      julia: 1.1
+      os: linux
+      script:
+        - julia --project=docs/ -e 'using Pkg; Pkg.develop(PackageSpec(path=pwd())); Pkg.instantiate()'
+        - julia --project=docs/ docs/make.jl
```

Doesn't work if you add
```suggestion
        - julia --project=docs/ --code-coverage=user docs/make.jl
```
?

---

I added this here https://github.com/QuantumBFS/YaoBlocks.jl/pull/28/commits/7968d20483ff0029e75018e5eaa8e2b8f776ef11

Is it correct? but it should show the printing of some types are covered at least... and it didn't appear in codecov

---

Seems to have worked on this branch when we tested: https://github.com/JuliaDocs/Documenter.jl/compare/mp/coverage https://codecov.io/gh/JuliaDocs/Documenter.jl/branch/mp%2Fcoverage

---

Ok... let me try again.